lib/media/media_source_engine.js

```diff
@@ -1707,6 +1707,45 @@ shaka.media.MediaSourceEngine = class {
     return this.mediaSource_.duration;
   }
 
+  /**
+   * Updates the live seekable range.
+   *
+   * @param {number} startTime
+   * @param {number} endTime
+   */
+  async setLiveSeekableRange(startTime, endTime) {
+    if (!('setLiveSeekableRange' in this.mediaSource_)) {
```

Comment:
I feel like this if clause (and the similar one in `clearLiveSeekableRange`) should probably be an assert instead.

We currently check to see if the appropriate API methods are supported before we call these methods, so it being an assertion won't make this PR work incorrectly.

And for future-proofing, I would rather have this fail loudly if we try to call it when the API is not available, than have it fail silently.